src/theme.js

```diff
@@ -91,7 +91,7 @@ export const gordonColors = {
 // Material-UI breakpoints. This is to allow components to use Material-UI's breakpoints since
 // you cannot import variables from a vars.scss to a JS file.
 export const windowBreakWidths = {
-  breakXS: 360,
+  breakXS: 0,
   breakSM: 600,
```

Comment:
I've realized that the XS breakpoint on Material-UI's Website was at 0 and not 360. I'm not sure why I had 360 there originally but I've changed it to match Material-UI's XS breakpoint